.env

```diff
@@ -1,4 +1,7 @@
 DB_HOST=remotemysql.com
 DB_USER=KI4VpRQy5l
 DB_PASS=kjuQsqSLer
-DB_NAME=KI4VpRQy5l
\ No newline at end of file
+DB_NAME=KI4VpRQy5l
+
+ACCESS_TOKEN_SECRET=0a8d8cdb0e06153c21cc13f7057bc65162ba2c0cf8b0163eb146da8314b469a794b6723a1bd98e96c5a99d3ab89adafb8a23f787e29ba460db8c4661128a1a16
+REFRESH_TOKEN_SECRET=7c01899130d42186164f28d8cd69d5b2c106c2513a0ca8d8409636e280f6f0e7995752295a3937fc49d05621786ae1baea977175e80437a2093f87db609300cc
```

Comment:
.env should be added to .gitignore, I asked

Reply:
solved